docs/developer-manual/consume.md

```diff
@@ -4,31 +4,248 @@ sidebar_position: 1
+Orcfax feeds into their dapp with all the gory (technical) details included.
 
-This page is being updated to reflect the upcoming changes in the Orcfax v1
-solution. Once this upgrade has been released, instructions on how to consume
-the enhanced datum will be made available here.
+## Overview
 
-Historical data concerning the v0 Orcfax solution can be found in
-[Archived resources](archived-resources) and information pertaining to policy
-ID's in [Policies](policies).
+### Orcfax
 
-## Fact statements
+Orcfax is a Cardano native oracle platform.
 
-A fact statement is a datum that the Orcfax network publishes to a Cardano
-blockchain transaction. This datum can then be used as an input to Cardano smart
-contracts and dApps.
+Orcfax is organized into feeds. Generally a feed concerns a time series data,
+such as the exchange rate from ADA to USD. A feed is a data pipeline that
+specifies how data is sourced, processed, and the format of its outputs.
 
-## UTxO model
+FIXME : See the orcfax explorer for more details on the feeds available and
+their definitions.
 
-A single fact statement is associated with a single [UTxO][utxo-1]. Each UTxO
-has a [reference input][reference-1] encoded as an [inline datum][datum-1].
+One part of the overall protocol is getting the data on-chain to be consumable
+by dapps and their end users. The part of a feed's output that ends up on-chain
+is called a **statement**. A statement is included in the datum of a transaction
+input, and thus available to dapps via the reference input field.
 
-Reference inputs are published using the Orcfax Protocol.
+### Orcfax-publish
 
-[utxo-1]: https://docs.cardano.org/learn/eutxo-explainer/
-[reference-1]: https://docs.cardano.org/about-cardano/evolution/upgrades/vasil/#reference-inputs-cip-31
-[datum-1]: https://docs.cardano.org/about-cardano/evolution/upgrades/vasil/#inline-datums-cip-32
+The part of the overall protocol getting the data on-chain is called
+**Orcfax-publish**. Orcfax-publish consists of three Plutus V2 scripts:
+
+1. FactStatement (FS)
+2. FactStatementPointer (FSP)
+3. Constitution (C)
+
+The FS script is the main script. It can be employed in both Mint and Spend
+purpose. It can mint only a single asset class we call FS token. An FS token can
+only exist at the FS script address (staking credentials are irrelevant here).
+The presence of an FS token in a UTXO verifies that the associated statement is
+authentic.
+
+Both the FSP and C scripts are simple updatable state scripts. That is, they
+allow a UTXO at the script address to hold a validity token and a piece of state
+(in the datum). When required, a twinned auth token is used to spend this UTXO
+and output another with the same validity token and a new state.
+
+In the case of the FSP, the state is simply the FS script hash. The FSP exists
+to allow Orcfax to update the FS script while allowing integrators to launch
+dapps without having to roll their own update mechanism. Orcfax reserves the
```

Comment:
"Orcfax reserves the ability to update the FS script should business or developmental needs require (eg in order to migrate to Plutus V3), and the FS script allows Orcfax to do this without impacting integrators."